meta-dts-distro/recipes-bsp/flashrom/flashrom_git.bb

```diff
@@ -6,9 +6,9 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
 
 DEPENDS = "pciutils libusb libftdi"
 
-BRANCH = "dasharo-release"
+BRANCH = "tgl_support"
```

Comment:
If we build from such a branch, I expect we will complain in a few days/weeks that the DTS build is broken because this branch does not exist. I think we need to have it merged first.

Reply:
https://github.com/Dasharo/flashrom/pull/16 is merged, so we can revert it to point to a new revision on dasharo-release branch

Reply:
Done https://github.com/Dasharo/meta-dts/compare/2729a4b1460cb694b602fefca187fed6073b48c1..36861b018683ee2fa74c83a5bee5f054169914bd